src/deepsparse/pipeline.py

```diff
@@ -696,10 +693,11 @@ class PipelineConfig(BaseModel):
             "specifies all available cores. Default is None"
         ),
     )
-    scheduler: str = Field(
-        default="async",
+    scheduler: Scheduler = Field(
+        default=Scheduler.single_stream,
```

Comment:
this is fine in controlled environments for getting the best single model numbers - also since we're always overriding with a context (elastic) in the server. however might want to think about how we can detect misuse and warn (ie initiating multiple single stream pipelines that are intended to run asynchronously )

Reply:
I can revert these changes from the PR - don't set the scheduler field of PipelineConfig anymore